apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -466,9 +472,33 @@ export class LandingPage extends ViewPlugin {
           <div class="d-flex justify-content-between">
             <div class="d-flex flex-column">
               <div class="border-bottom d-flex justify-content-between clearfix py-3 mb-4">
-                <div class="mx-4 w-100">
+                <div class="mx-4 w-100 d-flex">
                   ${img}
                   <audio id="remiAudio" muted=false src="assets/audio/remiGuitar-single-power-chord-A-minor.wav"></audio>
+                  <div class="w-80 pl-5 ml-5">
+                    <h6 class="mb-1">Quicklinks</h6>
+                    <spam class="pl-0">
+                      <u class="${css.text} pr-1" onclick=${() => migrateWorkspace()}>Migrate File system</u>
+                      to workspace
+                    </spam>
+                    <p class="font-weight-bold mb-0">2-step solution:</p>
+                    <li class="pl-1">
+                      manually
+                      <u class="${css.text} px-1" onclick=${() => downloadFiles()}>Download all Files</u>
+                      as a backup zip
+                    </li>
+                    <li class="pl-1">
+                      use
+                      <u class="${css.text} px-1" onclick=${() => startRestoreBackupZip()}>Restore Backup Zip</u>
+                      to current workspace
+                    </li>
+                    <p class="font-weight-bold mb-0 mt-2">Help:</p>
+                    <dir class="d-flex flex-column mt-1 pl-0">
+                      <a class="${css.text} mx-1" target="__blank" href=" https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c">Guide for migrating the File System to workspace</a>
```

Comment:
`Guide for migrating the File System to workspace` can be shortened as `Migration Guide`